migrations/0002_add_external_submissions.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.conf import settings
+import django.db.models.deletion
+import tracker.models.event
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tracker', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Runner',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('name', models.CharField(unique=True, max_length=64)),
+                ('stream', models.URLField(max_length=128)),
```

Comment:
While you're at it, you can get rid of the 'runneryoutube'/'runnertwitch'/'runnertwitter' fields on Donor as well.